lib/tooling/pvs-studio-tool.js

```diff
@@ -150,7 +150,6 @@ export class PvsStudioTool extends BaseTool {
 
     getDefaultExecOptions() {
         const execOptions = super.getDefaultExecOptions();
-        execOptions.env = process.env;
         execOptions.env.PATH = process.env.PATH + ':/opt/compiler-explorer/pvs-studio-latest/bin';
```

Comment:
.env is undefined here, should be first initialized as `{}`

Reply:
getDefaultExecOptions will always initialize env

Reply:
@partouf 's right - fixing